packages/html/src/spreadsheet/tests/spreadsheet-filter-menu.tsx

```diff
@@ -45,9 +49,9 @@ root.render(
                             </div>
                         </div>
                         <div className="k-columnmenu-item-wrapper">
-                            <div className="k-widget k-expander">
-                                <div className="k-columnmenu-item">
-                                    Filter by value
+                            <div className="k-expander">
+                                <div className="k-columnmenu-item k-focus">
```

Comment:
do not use states in the tests here, as they should be covered in the ColumnMenu tests already